scale/recipe/definition/json/definition_v6.py

```diff
@@ -31,6 +31,7 @@
             'type': 'string',
         },
         'input': INTERFACE_SCHEMA,
+        # 'input_conditon': DATA_FILTER_SCHEMA,
```

Comment:
Why are the filters commented out?

Reply:
That's a leftover that didn't get removed from my first idea of implementing this. Originally thought we would have needed conditions for each input, but this is not actually the case. I'll remove the commented out line.